cmd/monitor/main.go

```diff
@@ -36,15 +38,19 @@ import (
 	"github.com/cockroachdb/cockroach/util"
 	"github.com/cockroachdb/cockroach/util/log"
 	"github.com/cockroachdb/cockroach/util/retry"
+	"github.com/cockroachdb/cockroach/util/stop"
 )
 
-var outputDir = flag.String("dir", ".", "Directory in which to output the status logs to.")
-var interval = flag.Duration("interval", 10*time.Second, "Interval in which to poll the cluster's status.")
-var addr = flag.String("addr", ":26257", "The host:port of the cockroach cluster.")
+const (
+	// urlPath is the http path of the status server on each node.
+	urlPath = "_status/nodes"
+)
+
+var interval = flag.Duration("interval", 10*time.Second, "Interval in which to poll each services's status.")
+var addrs = flag.String("addrs", ":26257", "Comma-separated list of host:port status endpoints to monitor.")
```

Comment:
just "host:port to monitor". including "status endpoints" in the message makes it sounds like I should pass host:port/url